manuscript/Hacking.md

```diff
@@ -45,7 +47,12 @@ This generation of hackers will create technology to sell PT products around the
  
 The generation that was hacking Linux boxes while we were in university is the same generation that is solving fundamental problems today. 
 
-We need to make sure that this new generation knows what a Raspberry Pi is, knows what is inside, and knows how to build one. There is a factory in Portugal where this can be done.
+We need to make sure that this new generation knows what a Raspberry Pi is, knows what is inside, and knows how to build one (see https://www.raspberrypi.org/blog/ten-millionth-raspberry-pi-new-kit/). There should be a factory in Portugal that can do this.  
+
+In fact, there should be software and technology companies in every city and town in Portugal. Software development and application security lend themselves to remote work, with regular commutes to a central office for meetings etc.
+
+(See for example how many Portuguese security researchers participate in global/world wide bug bounties.)
```

Comment:
Remove brackets
the Evidence for from is at https://pages.bugcrowd.com/hubfs/PDFs/state-of-bug-bounty-2016.pdf

![image](https://cloud.githubusercontent.com/assets/656739/20941872/70d8d90c-bbf0-11e6-8789-cde2ce896cda.png)

Should we include this image in book?